app/models/group/externe_kontakte.rb

```diff
@@ -5,7 +5,11 @@
 #  or later. See the COPYING file at the top-level directory or at
 #  https://github.com/hitobito/hitobito_skv.
 
+class Group::ExterneKontakte < ::Group
 
-class Role::Leitung < ::Role
-  self.permissions = [:layer_full]
-end
\ No newline at end of file
+  class ExternerKontakt < ::Role; end
+  class SwissCanoePartner < ::Role; end
```

Comment:
Vielleicht sollte man den beiden Rollen noch `kind = :external` geben. Siehe auch: https://github.com/hitobito/hitobito/blob/master/doc/architecture/08_konzepte.md#group--and-roletypes (etwas weiter im Text versteckt)